apps/app_rpt.c

```diff
@@ -5514,6 +5514,10 @@ static int load_config(int reload)
 		ast_log(LOG_NOTICE, "Unable to open radio repeater configuration rpt.conf.  Radio Repeater disabled.\n");
 		return -1;
 	}
+	if (cfg == CONFIG_STATUS_FILEINVALID) {
```

Comment:
Not that it changes the code at all, but stylistically, in Asterisk modules this would be an else if chained off the previous if.